src/features/authentication/components/LoginForm/Categories/Categories.css

```diff
@@ -0,0 +1,46 @@
+.sport-card {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  border-radius: 30px; 
+  overflow: hidden;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+}
+
+.sport-image {
+  width: 100%;
+  height: 100%;
+  object-fit: cover; 
+  border-radius: 25px; 
+}
+
+.sport-overlay {
+  position: absolute;
+
+  bottom: 1px; 
+  left: 50%;
+  transform: translateX(-50%);
+  background: rgba(177, 179, 177, 0.71); 
+  color: white;
```

Comment:
colors should be css variables. In styles/colors.css But the color white there and access the colors with css variables